docs/website/docs/dlt-ecosystem/verified-sources/personio.md

```diff
@@ -0,0 +1,189 @@
+# Personio
+
+:::info Need help deploying these sources, or figuring out how to run them in your data stack?
+
+[Join our Slack community](https://join.slack.com/t/dlthub-community/shared_invite/zt-1n5193dbq-rCBmJ6p~ckpSFK4hCF2dYA)
+or [book a call](https://calendar.app.google/kiLhuMsWKpZUpfho6) with our support engineer Adrian.
+:::
+
+Personio is a human resources management software that helps businesses streamline HR processes,
+including recruitment, employee data management, and payroll, in one platform.
+
+Our Personio verified source loads data using Perosnio API to your preferred
+[destination](../destinations).
+
+:::tip 
+You can check out our pipeline example [here](https://github.com/dlt-hub/verified-sources/blob/master/sources/personio_pipeline.py). 
+:::
+
+Resources that can be loaded using this verified source are:
+
+| Name        | Description                                                                              |
+|-------------|------------------------------------------------------------------------------------------|
+| employees   | Retrieves company employees details. (Employees list, absense_entitlement, cost_centers) |
+| absences    | Retrieves list of various types of employee absences                                     |
+| attendances | Retrieves attendance records for each employee                                           |
+
+## Setup Guide
+
+### Grab credentials
+
+To load data from Personio, you need to API credentials, `client_id` and `client_secret`:
+
+1. Sign in to your Personio account, and ensure that your user account has API access rights.
+1. Navigate to Settings > Integrations > API credentials.
+1. Click on "Generate new credentials."
+1. Assign necessary permissions to credentials, i.e. read access.
+
+### Initialize the verified source
+
+To get started with your data pipeline, follow these steps:
+
+1. Enter the following command:
+
+   ```bash
+   dlt init personio duckdb
+   ```
+
+   [This command](../../reference/command-line-interface) will initialize
+   [the pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/personio_pipeline.py)
+   with Personio as the [source](../../general-usage/source) and [duckdb](../destinations/duckdb.md)
+   as the [destination](../destinations).
+
+1. If you'd like to use a different destination, simply replace `duckdb` with the name of your
+   preferred [destination](../destinations).
+
+1. After running this command, a new directory will be created with the necessary files and
+   configuration settings to get started.
+
+For more information, read the
+[Walkthrough: Add a verified source.](../../walkthroughs/add-a-verified-source)
```

Comment:
we periodically reorganize docs structure, so we don't have Walkthrough section anymore, we need to replace it with something neutral, at least: 
 For more information, read [Add a verified source.](../../walkthroughs/add-a-verified-source)

Reply:
done